spec/fixtures/germany_integration_test_cases.json

```diff
@@ -101,12 +101,12 @@
       {"bank_code":"51020000","account_number":"30009963","converted_bank_code":"50020200","converted_account_number":"30009963"}
     ]
   },
-  {
-    "convertor": "000900",
-    "valid": [
-      {"bank_code":"68351976","account_number":"1116232594","converted_bank_code":"68351557","converted_account_number":"3047232594"}
-    ]
-  },
+  // {
+  //   "convertor": "000900",
+  //   "valid": [
+  //     {"bank_code":"68351976","account_number":"1116232594","converted_bank_code":"68351557","converted_account_number":"3047232594"}
```

Comment:
Why have these been commented out? Are they no longer considered valid? I'm just concerned that we'll have examples of this in our DB already

Reply:
So the iban rules are still valid but no banks use them in the BLZ2 file

Reply:
Ok, so I think we can merge this but we should do a check (somehow) to verify we don't have any bank details registered using them. If somehow we do and they get sent to the network they will bounce so potentially we can reach out to Support to get details updated proactively.

Reply:
I still say we merge this however as it's better they fail before hitting the network (if they would fail, not 100% sure they would).